module/Decision/src/Decision/Mapper/Member.php

```diff
@@ -39,6 +39,25 @@ public function findByLidnr($number)
     }
 
     /**
+     * Finds members by (part of) their name.
+     *
+     * @param string $firstName (part of) the first name of a member
+     * @param string $lastName (part of) the last name of a member
+     *
+     * @return array
+     */
+    public function findByName($firstName, $lastName)
+    {
+        $qb = $this->getRepository()->createQueryBuilder('m');
+
+        $qb->where('m.firstName LIKE ?1')
+            ->orWhere('m.lastName LIKE ?2')
+            ->setParameter(1, $firstName . '%')
+            ->setParameter(2, '%' . $lastName . '%');
+
+        return $qb->getQuery()->getResult();
```

Comment:
This doesn't allow searching for full names. It might be better to solve this like I solved it in the GEWIS Database:

https://github.com/GEWIS/gewisdb/blob/master/module/Database/src/Database/Mapper/Member.php#L38

Also, limiting the number of results here would be nice as well.